packages/stacks/src/stacks.utils.ts

```diff
@@ -71,3 +73,110 @@ export function getStacksBurnAddress(chainId: ChainID): string {
       return 'ST000000000000000000002AMW42H';
   }
 }
+
+function cleanHex(hexWithMaybePrefix: string): string {
+  if (hexWithMaybePrefix !== 'string') return hexWithMaybePrefix;
+  return hexWithMaybePrefix.startsWith('0x')
+    ? hexWithMaybePrefix.replace('0x', '')
+    : hexWithMaybePrefix;
+}
+
+export function hexToBuff(hex: string): Buffer {
+  return Buffer.from(cleanHex(hex), 'hex');
+}
+
+export const stacksValue = ({
+  value,
+  fixedDecimals = true,
+  withTicker = true,
+  abbreviate = false,
+}: {
```

Comment:
This really old code that'd be best not copied across. Can we refactor it out entirely?